gcsstore.go

```diff
@@ -308,6 +308,10 @@ func (o *gcsFSObject) Open(accesslevel AccessLevel) (*os.File, error) {
 			}
 			defer rc.Close()
 
+			if _, err := cachedcopy.Seek(0, os.SEEK_SET); err != nil {
+				return nil, fmt.Errorf("error seeking to start of cachedcopy err=%v", err) //don't retry on local fs errors
+			}
+
 			_, err = io.Copy(cachedcopy, rc)
 			if err != nil {
 				errs = append(errs, fmt.Errorf("error coping bytes. err=%v", err))
```

Comment:
GCS does crc32 checking on the file.  So if it's successful it will be ok.
The corruption came in that we were appending to a partially downloaded file.